src/errors/validation_exception.rs

```diff
@@ -123,38 +170,61 @@ impl<'a> IntoPy<ValError<'a>> for ValidationError {
 
 #[pymethods]
 impl ValidationError {
+    #[pyo3(signature = (message, errors, error_mode = None, loc_prefix = None))]
     #[new]
-    fn py_new(title: PyObject, line_errors: &PyList, error_mode: Option<&str>) -> PyResult<Self> {
+    fn py_new(
+        message: PyObject,
+        errors: &PyList,
+        error_mode: Option<&str>,
+        loc_prefix: Option<&PyAny>,
+    ) -> PyResult<Self> {
         Ok(Self {
-            line_errors: line_errors.iter().map(PyLineError::try_from).collect::<PyResult<_>>()?,
-            title,
-            error_mode: ErrorMode::try_from(error_mode)?,
+            exceptions: errors.iter().map(WrappedError::extract).collect::<Result<_, _>>()?,
+            message,
+            error_mode: error_mode.try_into()?,
+            loc_prefix: loc_prefix.try_into()?,
         })
     }
 
     #[getter]
-    fn title(&self, py: Python) -> PyObject {
-        self.title.clone_ref(py)
+    fn title(&self) -> PyObject {
+        self.message.clone()
```

Comment:
```suggestion
    fn title(&self, py: Python) -> PyObject {
        self.message.clone_ref(py)
```
I think it should be faster.